README.rst

```diff
@@ -406,12 +406,12 @@ To register a signal you need to include following code in your ``apps.py``.
         from openwisp_notifications.handlers import register_notification_cache_update
 
         model = load_model('app_name', 'model_name')
-        register_notification_cache_update(model, post_save, signal_uid="myapp_mymodel_notification_cache_invalidation")
+        register_notification_cache_update(model, post_save, dispatch_uid="myapp_mymodel_notification_cache_invalidation")
 
 **Note**: You need to import ``register_notification_cache_update`` inside the ``ready`` function or 
 you can define another funtion to register signals which will be called in ``ready`` and then it will be 
-imported in this function. Also ``signal_uid`` is unique identifier of a signal. You can pass any 
-value you want but it is suggested to pass signal name as a string. By default it is ``None``.
+imported in this function. Also ``dispatch_uid`` is unique identifier of a signal. You can pass any 
+value you want but it needs to be unique. For more details read `preventing-duplicate-signals <https://docs.djangoproject.com/en/dev/topics/signals/#preventing-duplicate-signals>`_
```

Comment:
I think it is better that we explicitly tell users that they will be going to an external website by clicking on this link. 

```suggestion
value you want but it needs to be unique. For more details read `preventing duplicate signals section of Django documentation<https://docs.djangoproject.com/en/dev/topics/signals/#preventing-duplicate-signals>`_
```